pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -9,46 +8,47 @@ module.exports = {
-    'Please include this file if you are submitting a bug report.',
+      'You can download server logs to make the problem detection simpler. ' +
+      'Please include this file if you are submitting a bug report.',
   alertManager: {
     ip: process.env.VM_IP,
     service: ':9093/#/alerts',
     rule:
-      'groups:\n' +
-      '  - name: AutoTestAlerts\n' +
-      '    rules:\n' +
-      '    - alert: InstanceDown\n' +
-      '      expr: up == 0\n' +
-      '      for: 20s\n' +
-      '      labels:\n' +
-      '        severity: critical\n' +
-      '      annotations:\n' +
-      '        summary: "Instance {{ $labels.instance }} down"\n' +
-      '        description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 20 seconds."',
+        'groups:\n' +
+        '  - name: AutoTestAlerts\n' +
+        '    rules:\n' +
+        '    - alert: InstanceDown\n' +
+        '      expr: up == 0\n' +
+        '      for: 20s\n' +
+        '      labels:\n' +
+        '        severity: critical\n' +
+        '      annotations:\n' +
+        '        summary: "Instance {{ $labels.instance }} down"\n' +
+        '        description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 20 seconds."',
     editRule:
-      'groups:\n' +
-      '  - name: AutoTestAlertsEdited\n' +
-      '    rules:\n' +
-      '    - alert: InstanceDown\n' +
-      '      expr: up == 0\n' +
-      '      for: 60s\n' +
-      '      labels:\n' +
-      '        severity: critical\n' +
-      '      annotations:\n' +
-      '        summary: "Instance {{ $labels.instance }} down"\n' +
-      '        description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 20 seconds."',
+        'groups:\n' +
+        '  - name: AutoTestAlertsEdited\n' +
+        '    rules:\n' +
+        '    - alert: InstanceDown\n' +
+        '      expr: up == 0\n' +
+        '      for: 60s\n' +
+        '      labels:\n' +
+        '        severity: critical\n' +
+        '      annotations:\n' +
+        '        summary: "Instance {{ $labels.instance }} down"\n' +
+        '        description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 20 seconds."',
     ruleName: 'AutoTestAlerts',
     editRuleName: 'AutoTestAlertsEdited',
   },
   messages: {
     successPopUpMessage: 'Settings updated',
     invalidDataDurationMessage: 'Value should be in range from 1 to 3650',
+    invalidDataDurationPopUpMessage: 'data_retention: should be a natural number of days',
     requiredFieldMessage: 'Required field',
     invalidSSHKeyMessage: 'Invalid SSH key.',
     successAlertmanagerMessage: 'Alert manager settings updated',
     invalidAlertmanagerMissingSchemeMessage:
-      'Invalid alert_manager_url: invalid_url - missing protocol scheme.',
+        'Invalid alert_manager_url: invalid_url - missing protocol scheme.',
```

Comment:
is the spacing changed?

Reply:
It actually corresponds to the .editorconfig
indent_style = space
indent_size = 2

Reply:
oh, for some reason it was not applied correctly, i'll fix it